From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Aleksei Kuzmin <alkuzmin@microsoft.com>
Date: Thu, 20 Sep 2018 17:50:45 -0700
Subject: exclude-a-few-test-files-from-build.patch

Compilation of those files fails with the Chromium 68.
Remove the patch during the Chromium 69 upgrade.

diff --git a/third_party/blink/renderer/platform/BUILD.gn b/third_party/blink/renderer/platform/BUILD.gn
index ba938f7d75072c6d15de08528d6b1cac94ecdbb1..5878cbf7767adad6365d02e389b8b4a1774a3539 100644
--- a/third_party/blink/renderer/platform/BUILD.gn
+++ b/third_party/blink/renderer/platform/BUILD.gn
@@ -1737,7 +1737,7 @@ jumbo_source_set("blink_platform_unittests_sources") {
     "graphics/paint/drawing_display_item_test.cc",
     "graphics/paint/drawing_recorder_test.cc",
     "graphics/paint/float_clip_rect_test.cc",
-    "graphics/paint/geometry_mapper_test.cc",
+    #"graphics/paint/geometry_mapper_test.cc",
     "graphics/paint/geometry_mapper_transform_cache_test.cc",
     "graphics/paint/paint_chunk_test.cc",
     "graphics/paint/paint_chunker_test.cc",
